Plugin/UIScopeLabel.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace AvS\ScopeHint\Plugin;
+
+use Magento\Eav\Api\Data\AttributeInterface;
+use Magento\Framework\Stdlib\ArrayManager;
+use Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Eav;
+
+class UIScopeLabel
+{
+    public function __construct(
+        protected ArrayManager $arrayManager,
```

Comment:
Can you make that PHP 7.4 compatible? And private instead of protected, or are there good reasons for protected?

Reply:
Done. No special reason.